ivadomed/__init__.py

```diff
@@ -1 +1,2 @@
-__version__ = "2.2.1"
```

Comment:
is this change still compatible with RTD build (which fetches the version, i think..?)

Reply:
Looks ok https://readthedocs.org/projects/ivado-medical-imaging/builds/11961354/